sympy/functions/special/bessel.py

```diff
@@ -464,6 +464,8 @@ def eval(cls, nu, z):
             return S.Zero
 
         if nu.is_integer:
+            if z is S.Infinity:
```

Comment:
It is not necessary that nu be an integer, the limit at oo is 0 for all nu.

Reply:
Yeah, will make the required change.